src/client/helpers/socialProfiles.js

```diff
@@ -17,6 +18,7 @@ export const getTwitterShareURL = (text, url) =>
 export default [
   { id: 'facebook', icon: 'facebook', color: '#3b5998', name: 'Facebook' },
   { id: 'twitter', icon: 'twitter', color: '#00aced', name: 'Twitter' },
+  { id: 'linkedin', icon: 'linkedin', color: '#007bb6', name: 'Linkedin' },
```

Comment:
The proper name is `LinkedIn` not `Linkedin`

Reply:
True, I've even checked on the Logo, to see that, but the title on the page agrees with you. gonna change it right now thanks